README.md

```diff
@@ -296,44 +296,43 @@ For the complete code see [examples/libenvpp_range_example.cpp](examples/libenvp
 
 ### Option Variables
 
-Another frequent use-case is that a value is one of a given set of options. For this an environment variable can be registered with `register_[required]_option`, which takes a list of valid options, against which the value is checked. For example:
+Another frequent use-case is that a value is one of a given set of options. For this an environment variable can be registered with `register_[required]_option`, which takes a list of either pairs of strings and corresponding options, or just valid options, against which the value is checked. For example:
 
 ```cpp
 enum class option {
-    first_choice,
-    second_choice,
-    third_choice,
-    default_choice,
+    first,
+    second,
+    default,
```

Comment:
`default` is a keyword!

Reply:
That's what I get for not copy-pasting the example code from the actual example but trying to simplify it for the readme :P